modules/flowable-cmmn-converter/src/main/java/org/flowable/cmmn/converter/export/ReactivationEventListenerExport.java

```diff
@@ -48,6 +49,15 @@ protected void writePlanItemDefinitionSpecificAttributes(ReactivateEventListener
                 CmmnXmlConstants.ATTRIBUTE_EVENT_LISTENER_AVAILABLE_CONDITION,
                 reactivationEventListener.getAvailableConditionExpression());
         }
+
+        if (StringUtils.isNotEmpty(reactivationEventListener.getEventType()) && reactivationEventListener.getExtensionElements().get("eventType") == null) {
```

Comment:
Why do we need this change? there is already an exported flowable attribute with listener type `reactivate` on line 45. Perhaps the JSON conversion should use that instead.